From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Brad Miller <brad@bradhouse.com>
Date: Sat, 6 Sep 2014 18:41:05 -0400
Subject: [PATCH 0335/6262] Change the tail command for capturing the logs on
 deployment to check the file every second instead of continuously to avoid
 the 100% CPU time problem from artf3524

---
 .../src/main/resources/cpp-zip/ant/build.xml                    | 2 +-
 .../src/main/resources/java-zip/ant/build.properties            | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
index 68583c6534cd6e54324b025cd9ec5f94f9da6900..36384450d6659305df89ab9e7186aac164ea0d13 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
@@ -46,7 +46,7 @@
        username="${username}"
        password="${password}"
        trust="true"
-       command="tail -F -s 0 -n 0 ${deploy.log.file}"/>
+       command="tail -F -n 0 ${deploy.log.file}"/>
   </target>
   
   <target name="kill-program" depends="get-target-ip" description="Kill the currently running FRC program">
diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties b/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties
index 7c800ce92d30d61f603a2bc7d7c7d7bc5da97f89..5d2c4701bfacd5e9b4ce7742f1c8a75a48b213e6 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties
@@ -4,7 +4,7 @@ password=
 deploy.dir=/home/admin
 deploy.kill.command=. /etc/profile.d/natinst-path.sh; /usr/local/frc/bin/frcKillRobot.sh -t -r
 deploy.log.file=/var/local/natinst/log/FRC_UserProgram.log
-deploy.log.command=tail -F -s 0 -n 0 ${deploy.log.file}
+deploy.log.command=tail -F -n 0 ${deploy.log.file}
 debug.flag.dir=/tmp/
 command.dir=/home/lvuser/
 
